demos/smartlab_demo/python/smartlab_demo.py

```diff
@@ -0,0 +1,161 @@
+"""
+ Copyright (C) 2021-2022 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+import time
+from collections import deque
+from argparse import ArgumentParser, SUPPRESS
+from object_detection.detector import Detector
+from segmentor import Segmentor, SegmentorMstcn
+from evaluator import Evaluator
+from display import Display
+# from openvino.runtime import Core
+from openvino.inference_engine import IECore as Core
+
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                    help='Show this help message and exit.')
+    args.add_argument("-d", "--device", type=str, default='CPU', required=False,
+                        help="Optional. Specify the target to infer on CPU or GPU.")
+    args.add_argument('-tv', '--topview', required=True,
+                    help='Required. Topview stream to be processed. The input must be a single image, '
+                    'a folder of images, video file or camera id.')
+    args.add_argument('-fv', '--frontview', required=True,
+                    help='Required. FrontView to be processed. The input must be a single image, '
+                    'a folder of images, video file or camera id.')
+    args.add_argument('-m_ta', '--m_topall', help='Required. Path to topview all class model.', required=True, type=str)
+    args.add_argument('-m_tm', '--m_topmove', help='Required. Path to topview moving class model.', required=True, type=str)
+    args.add_argument('-m_fa', '--m_frontall', help='Required. Path to frontview all class model.', required=True, type=str)
+    args.add_argument('-m_fm', '--m_frontmove', help='Required. Path to frontview moving class model.', required=True, type=str)
+    args.add_argument('--mode', default='multiview', help='Optional. action recognition mode: multiview or mstcn', type=str)
```

Comment:
I suppose you expect the following behavior: if the --mode is set as multiview, then the user must pass `-m_en`, `m_de` arguments, otherwise the --mode is mstcn and the user must pass `-m_i3d`, `-m_mstcn`.  
 Unfortunately it doesn't work and therefore the tests are not passed. You need to modify the parser correctly to provide such behavior.